samples/boards/nrf52/power_mgr_os/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _nrf52-power-mgr-sample:
```

Comment:
This looks like it's a copy of an existing sample documentation with no obvious changes (including the title):  ``/samples/boards/nrf52/power_mgr/README.rst``.    The doc build complained that the label at the top of the doc here was the same as another document.

Reply:
yes David, will fix it.